lib/deploy.js

```diff
@@ -41,9 +44,17 @@ function sftpDeploy(opts) {
         // Wait for the transfer to finish...
         rstdin.once('finish', function() {
           tessel.logs.info('Deployed.');
+          if (push) {
+            tessel.logs.info('Setting up to run on boot...');
+            conn.exec("echo '#!/bin/sh\nnode app' > " + filepath + "/start && chmod +x '" + filepath + "/start'", function (err, newStream) {
```

Comment:
You also probably want to do something like

```
cd /app
exec node .
```
to give the program a sane working directory.